### PR TITLE
feat: add isLocal to verb builders

### PR DESCRIPTION
**Pull request by sitaram-kalluri**
**- What I did**
- Add isLocal to UpdateVerbBuilder and local_lookup_verb builder

**- How I did it**

**- How to verify it**

**- Description for the changelog**
- Add isLocal to verb builder

### CODE REVIEW

File: at_commons/lib/src/verb/update_verb_builder.dart [hunk 1]
```diff
@@ -142,13 +146,15 @@ class UpdateVerbBuilder implements VerbBuilder {
     if (encoding != null && encoding!.isNotEmpty) {
       command += '$ENCODING:$encoding';
     }
+    if (isLocal) {
+      command += 'local:';
+    }
     if (isPublic) {
       command += 'public:';
```
Comment (gkc) [inline at_commons/lib/src/verb/update_verb_builder.dart:157]: We need to make sure there are guard conditions which prevent local keys also being public. A local key **must not** be (1) public (2) shared with anyone else. Likewise, a key that is public, or shared with someone else, **must not** be local

**Review by gkc — CHANGES_REQUESTED** [review]
LGTM, one request

File: at_commons/lib/src/verb/llookup_verb_builder.dart [hunk 2]
```diff
@@ -31,12 +31,19 @@ class LLookupVerbBuilder implements VerbBuilder {
...
+  /// If the key is local, the key does not sync between cloud and local secondary
+  bool isLocal = false;
+
   @override
   String buildCommand() {
     var command = 'llookup:';
     if (operation != null) {
       command += '$operation:';
     }
+    if(isLocal){
```
Comment (murali-shris) [inline at_commons/lib/src/verb/llookup_verb_builder.dart]: dart format missing

**Review by murali-shris — APPROVED** [review]

**Review by gkc — APPROVED** [review]
LGTM for now. Could do with adding more validation of mutual exclusivity in the AtKey setters

**Pull request merged**